.github/workflows/zowe-versions_updateVersions_createPR.yml

```diff
@@ -73,7 +73,8 @@ jobs:
           branch_name="v$version_name/master"
           pr_title="Update zowe component versions for $version_name"
           body_file=description.txt
-          pr_number=$(gh pr create --base master --head "$branch_name" --title "$pr_title" --body-file "$body_file") | sed 's/.*\/\([0-9]\+\)$/\1/'
+          pr_url=$(gh pr create --base master --head "$branch_name" --title "$pr_title" --body-file "$body_file")
+          pr_number=pr_url | sed 's/.*\/\([0-9]\+\)$/\1/'

```

Comment:
Did you test this command in Bash locally?
I believe in order to pipe `pr_url` to "sed" we may need to reference it with a dollar sign and the "echo" command.
Also the command expression needs to be wrapped inside `$(` `)`.

Something like this: `pr_number=$(echo $pr_url | sed ...)`